vnext/Microsoft.ReactNative/Modules/WebSocketModuleUwp.cpp

```diff
@@ -164,7 +165,25 @@ void WebSocketModule::WebSocket::connect(
 
   if (!SUCCEEDED(hr)) {
     winrt::hresult_error e{hr};
-    OutputDebugString("WebSocket.connect failed (0x%8X) %ls\n", e);
+    std::wstring error;
+#define INET_ERR_OUT_FORMAT_BUFFER_SIZE 256
+    const DWORD err = hr & 0xffff;
+    if (err >= INTERNET_ERROR_BASE && err <= INTERNET_ERROR_LAST) {
+      wchar_t errorStr[INET_ERR_OUT_FORMAT_BUFFER_SIZE] = {};
```

Comment:
Is this change related to suppressing exception in favor of HRESULT?
Does this code retrieve the error message originally provided with the exception?